test/passing/infix_bind.ml

```diff
@@ -23,27 +23,37 @@ f x
 ;;
 f x >>= fun y ->
 g y >>= fun () ->
-f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
+f x >>= fun y ->
+g y >>= fun () ->
+f x >>= fun y ->
+g y >>= fun () ->
+y ()
 
 ;;
 f x >>= function
 | A -> (
     g y >>= fun () ->
     f x >>= fun y ->
     g y >>= function
-    | x -> ( f x >>= fun y -> g y >>= function _ -> y () ) )
+    | x -> (
+        f x >>= fun y ->
+        g y >>= function
+        | _ -> y () ) )
 
 ;;
-eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee |> fun x -> x
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee |> fun x ->
+x
```

Comment:
Would it make sense to add an exception for these cases ?
It seems a bit too verbose when there is only one bind.

Reply:
The problem is you can't just put this into an `hvbox` otherwise we have the current behavior where the last infix-bind doesn't break if it fits on 1 line.

Reply:
I see. It would be too complicated to detect the "chain" of bind operators ?

Reply:
It's not impossible to do, but I'm not sure it is worth implementing such a specific case. I will have a look on what it looks like on real code.